ember-cli-build.js

```diff
@@ -0,0 +1,49 @@
+var Funnel = require('broccoli-funnel');
+var Rollup = require('broccoli-rollup');
+var JSHint = require('broccoli-jshint');
+var MergeTrees = require('broccoli-merge-trees');
+var concat = require('broccoli-concat');
+
+module.exports = function(/* defaults */) {
+  var src = new Funnel('src', {
+    include: ['**/*.js'],
+    destDir: '/src'
+  });
+
+  var bundled = new Rollup(src, {
+    rollup: {
+      entry: 'src/index.js',
+
+      sourceMap: true,
```

Comment:
output es as well as UMD, add a module and js:next key so library can be rolled up or formatted differently.